docs/configuration/docker_deployment.md

```
@@ -0,0 +1,149 @@
+      - stalwart-data:/opt/stalwart
+
+volumes:
+  stalwart-data:
+```
+
+### Troubleshooting
+
+#### OIDC Discovery Errors to localhost
+
+**Symptom**: Browser console shows errors like:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)